Onboarding updates to storage monitoring (#31928)
DataDog/documentation: content/en/integrations/guide/storage-monitoring-setup.md

              }
        ```

>     2. Under "Enable it for me", select the regions or accounts you want to enable and assign a destination bucket per region or per account to store S3 Inventory reports. You can either use an existing bucket or create one in AWS.

Reviewer: ```suggestion
    2. On the **Enable it for me** tab, select the regions or accounts you want to enable and assign a destination bucket per region or per account to store S3 Inventory reports. You can either use an existing bucket or create one in AWS.
```

   - **SourceBucketPrefix**: This parameter limits the inventory generation to a specific prefix in the source bucket
   - **DestinationBucketPrefix**: If you want to reuse an existing bucket as the destination, this parameter allows the inventory files to be shipped to a specific prefix in that bucket. Ensure that any prefixes do not include trailing slashes (`/`)

    {{< img src="integrations/guide/storage_monitoring/bucket_policy_stack_details.png" alt="Stack parameters for bucket policy" responsive="true" style="width:90%;" >}}

6. On the **Review and create** step, verify the parameters have been entered correctly, and click **Submit**.

### Finish setting up S3 buckets for Storage Monitoring
  After completing the CloudFormation setup, enable buckets for Storage Monitoring from the Datadog UI:
  - Navigate to **Storage Monitoring** → [Enable Buckets][105]

>   - In Step 2: "Enable S3 Inventory to get prefix level monitoring", select **"I enabled it myself"**

Reviewer: ```suggestion
  - In Step 2: **Enable S3 Inventory to get prefix level monitoring**, select _I enabled it myself_
```

>   - Choose the destination buckets that contain the inventory files for the source buckets you want to monitor and click Confirm

Reviewer: ```suggestion
  - Choose the destination buckets that contain the inventory files for the source buckets you want to monitor and click **Confirm**
```

### Finish setting up S3 buckets for Storage Monitoring

  After the inventory configuration is set up and your inventory files begin appearing in the destination bucket, enable buckets for Storage Monitoring from the Datadog UI:
  - Navigate to **Storage Monitoring** → [Enable Buckets][405]

>   - In Step 2: "Enable S3 Inventory to get prefix level monitoring", select **"I enabled it myself"**

Reviewer: ```suggestion
  - In Step 2: **Enable S3 Inventory to get prefix level monitoring**, select _I enabled it myself_
```

>   - Choose the destination buckets that contain the inventory files for the source buckets you want to monitor and click Confirm

Reviewer: ```suggestion
  - Choose the destination buckets that contain the inventory files for the source buckets you want to monitor and click **Confirm**
```

### Post-setup steps

  After the inventory configuration is set up and your inventory files begin appearing in the destination bucket, enable buckets for Storage Monitoring from the Datadog UI:
  - Navigate to **Storage Monitoring** → [Enable Buckets][205]

>   - In Step 2: "Enable S3 Inventory to get prefix level monitoring", select **"I enabled it myself"**

Reviewer: ```suggestion
  - In Step 2: **Enable S3 Inventory to get prefix level monitoring**, select _I enabled it myself_
```

  - Choose the destination buckets that contain the inventory files for the source buckets you want to monitor and click Confirm

{{< img src="integrations/guide/storage_monitoring/enabled-it-myself.png" alt="Select destination buckets for enabling Storage Monitoring" responsive="true">}}

[201]: https://console.aws.amazon.com/s3/bucket/create
[202]: https://docs.aws.amazon.com/AmazonS3/latest/userguide/add-bucket-policy.html
[203]: https://console.aws.amazon.com/s3/buckets
[204]: https://aws.amazon.com/s3/pricing/
[205]: https://app.datadoghq.com/storage-monitoring?mConfigure=true&mStorageRecGroupBy=&mView=s3
[206]: https://docs.aws.amazon.com/AmazonS3/latest/userguide/configure-inventory.html
{{% /tab %}}

{{% tab "Existing S3 Inventory" %}}

  If you have already configured S3 Inventory for the buckets you want to monitor, enable buckets for Storage Monitoring from the Datadog UI:
      - Navigate to **Storage Monitoring** → [Enable Buckets][105]

>       - In Step 2: "Enable S3 Inventory to get prefix level monitoring", select **"I enabled it myself"**

Reviewer: ```suggestion
  - In Step 2: **Enable S3 Inventory to get prefix level monitoring**, select _I enabled it myself_
```
